text/0249-v2-experiments.md

```diff
@@ -67,7 +67,13 @@ levels of abstraction. Changing to vending the Construct Library as a monolithic
 package is one part of making that possible; we should make sure our approach to
 unstable code also takes this into account.
 
-## 3. The CDK team can still perform API experiments
+## 3. We want to encourage customers to not declare a fixed version on packages we own, and are intended to be widely used.
```

Comment:
I disagree, I think that this should be a goal.

> and in fact, with mechanisms like package-lock.json and similar ones, the customer effectively depends on a fixed version anyway

But they have tools such as dependebot that will update their lock file according to the declared version in the `package.json` file. It is a very different approach than using fixed version in a `package.json` files, which basically means "I only ever want to upgrade to a newer version of this package manually". 

This goal is the motivation behind option 6, so I think we should agree on it first.

Reply:
I think this is not a goal, but simply a consequence of the observation that "we can't release breaking changes in new versions of `aws-cdk-lib`".

But I honestly don't care that much, do what you feel is right 🙂 (if you're going to leave it, maybe use the phrasing I suggested above, because right now it reads pretty weird 😬).

Reply:
Removed, I left it in the advantages section for option 6.